WSRequestHandler.cpp

```diff
@@ -401,8 +413,90 @@ void WSRequestHandler::HandleStartStreaming(WSRequestHandler* req)
 {
 	if (obs_frontend_streaming_active() == false)
 	{
+		obs_data_t* streamData = obs_data_get_obj(req->data, "stream");
```

Comment:
It also allows you to have a separate configuration specifically just for the web socket client so that UI users would not see the changes to the settings and that if they started streaming they would remain with the original streaming server settings.